tsconfig.json

```diff
@@ -1,13 +1,14 @@
 {
   "extends": "./config/tsconfig.base.json",
   "include": [
-    "src/**/*.ts",
+    "src/**/*.ts"
   ],
   "compilerOptions": {
     "outDir": "build",
     "declaration": true,
     "experimentalDecorators": true,
     "incremental": true,
+    "target": "es6"
```

Comment:
target: es2018 is already in tsconfig.base.json; why are you overriding it here?

Reply:
Thanks, hadn't realized it was there. Ran into some linting issues and the 'top' fix was to state the es target explicitly. I'll look into it again.

Reply:
Ah, its a configuration problem with the Language Server I'm using for typescript, not our project's config. Thanks